app/src/main/java/com/kickstarter/viewmodels/projectpage/AddOnsViewModel.kt

```diff
@@ -114,13 +76,9 @@ class AddOnsViewModel(val environment: Environment) : ViewModel() {
                     apolloClient.getShippingRules(
                         reward = reward
                     ).subscribe { shippingRulesEnvelope ->
```

Comment:
There is still a subsciption and disposable being used here. We should not use RXJava, but `.asFlow` and use flow streams.
![Screenshot 2024-05-28 at 5 50 44 PM](https://github.com/kickstarter/android-oss/assets/4083656/858b6761-e022-44f0-ae36-1b3dd03719e5)

[Take a look to this document in case of doubt ](https://kickstarter.atlassian.net/wiki/spaces/NT/pages/2602729524/Coroutines+KS+Best+practices#Consume-Network-Layer-still-using-RXJava2)

Reply:
Oops good catch. Thanks Isa.